config/initializers/delayed_job.rb

```diff
@@ -1,4 +1,3 @@
-Delayed::Worker.logger = Logger.new(Rails.root.join('log', 'delayed_job.log'))
```

Comment:
where are these logs going to be written now?

Reply:
I would have liked to find a better solution for this, but it was a bit tricky!

In _theory_, DelayedJob should create a new logger instance when it forks, and it should be logging to `log/delayed_job.log` by default [here](https://github.com/collectiveidea/delayed_job/blob/baed6e813870e1144e7a4291bc71e06a67a533de/lib/delayed/command.rb#L133). But it's actually logging to the main Rails log (`log/<environment>.log`).

I've tried many different options but ultimately: any time this line in the delayed job initializer it ends up throwing fatal errors whenever the anything touches the Rails cache inside of a running job. I think there's some kind of clash between the cache instrumentation and the delayed_job instrumentation, resulting in errors like this: 
```
 Redis::CommandError · ERR unknown command `I,`, with args beginning with: `[2021-05-21`, `03:13:22#32129]`, `INFO`, `--`, `:`, `[ActiveJob]`, `[ConfirmOrderJob]`, `[f38147f0-dbf1-45cc-9206-e880a4c3585a]`, 
```
the arguments that should be being passed to the log are mistakenly being passed to the cache client. If you look in the full stack trace for [the errors](https://app.bugsnag.com/open-food-france/coopcircuits/errors/60a634609010590007701d6f?filters%5Bevent.since%5D%5B0%5D=30d&filters%5Berror.status%5D%5B0%5D=open), both `DelayedJob` and the Rails cache are wrapping the action inside of an `ActiveSupport::Instrumenter` to write to the logs whilst performing the operation, and it seems like the logger instance set on `DelayedJob` is ultimately not behaving properly, possibly due to the way file descriptors are handled when the process forks in daemonized mode?

If anyone else can make this work, I'd appreciate it! Ideally we'd return to the previous behavior where we write to a separate log file, but without everything bowing up. :boom: